testsuite/unittests/test_lossy_synapse.sli

```diff
@@ -0,0 +1,105 @@
+/*
+ *  test_lossy_connection.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/* Testscript 
+
+Name: testsuite::test_lossy_synapse - test of lossy_synapse connection
+
+Synopsis: (test_lossy_synapse) run -> measure average number of spikes transmitted by a lossy synapse
+
+Description:
+1000 spikes generated by spike_generator are sent to parrot neurons using a lossy_synapse with transmission probability 50%. 
+If the average number of spikes detected by spike detectors equals to 500 (with marginal error 5%), the synapse works fine.
+Also tests the interval of possible probabilities, [0, 1].
+
+Author: Maximilian Schmidt, Milena Menezes Carvalho, 2017
+*/
+
+(unittest) run
+/unittest using
+
+% test parameters -------------------------------------------------
+/N_spikes  1000 def  % Number of spikes send via the lossy synapse to each parrot neuron
+/n_parrots 10 def    % Number of parrot neurons receiving spikes
+/p 0.5 def           % Transmission probability of the lossy synapse
+/margin 0.05 def     % allowed relative deviation of measured mean from the target mean
+% test routine ----------------------------------------------------
+
+/testrun
+{
+  /seed Set
+  ResetKernel
+  0 << /rng_seeds [seed] >> SetStatus
+
+  % build
+  /spike_generator Create /sg Set
+  /parrot_neuron 10 Create 
+  [2 11] Range /pn Set
+  /spike_detector Create /sd Set
+
+  % spike_generator 
+  sg << /spike_times [1. N_spikes cvd] Range >> SetStatus
+
+  % connects spike_generator to all parrot neurons via lossy_synapse with transmission
+  % probability p
+  [sg] pn << /rule /all_to_all >> << /model /lossy_synapse /p_transmit p >> Connect
+
+  % connects parrot neurons to spike_detector
+  pn [sd] Connect
+
+  % simulates for 1010ms
+  N_spikes 10. add Simulate
```

Comment:
while we were testing we wanted to make sure that the simulation had enough time to record all spikes and quit successfully, but 10ms does seem like too much right now. thanks for the input!